protocol/keepalive/README.md

```
@@ -0,0 +1,117 @@
+
+## Protocol Identifiers
+
+| Property | Value |
+|----------|-------|
+| Protocol Name | `keep-alive` |
+| Protocol ID | `8` |
+| Mode | Node-to-Node |
+
+## State Machine
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)